functests/5_latency_testing/5_latency_testing_suite_test.go

```diff
@@ -26,32 +31,58 @@ var prePullNamespace = &corev1.Namespace{
 		Name: "testing-prepull",
 	},
 }
+var initialIsolated, initialReserved *performancev2.CPUSet
+var profileNeedsUpdate bool = false
 
-var _ = AfterSuite(func() {
-	prePullNamespaceName := prePullNamespace.Name
-	err := testclient.Client.Delete(context.TODO(), prePullNamespace)
-	testlog.Infof("deleted namespace %q err=%v", prePullNamespace.Name, err)
-	Expect(err).ToNot(HaveOccurred())
-	err = namespaces.WaitForDeletion(prePullNamespaceName, 5*time.Minute)
-})
+var _ = BeforeSuite(func() {
+	if !testclient.ClientsEnabled {
+		testlog.Errorf("client not enabled")
+	}
 
-func Test5LatencyTesting(t *testing.T) {
-	RegisterFailHandler(Fail)
+	//update PP isolated CPUs. the new cpu set for isolated should have odd number of CPUs to avoid failing teh pod on SMTAlignment error,
+	// and should be greater than what is requested by the test cases in the suite so the test runs properly
+	profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+	Expect(err).ToNot(HaveOccurred())
 
-	if !testclient.ClientsEnabled {
-		t.Fatalf("client not enabled")
+	initialIsolated = profile.Spec.CPU.Isolated
+	initialReserved = profile.Spec.CPU.Reserved
+	//updated both sets to ensure there is no overlap
+	latencyIsolatedSet := performancev2.CPUSet("1-9")
+	latencyReservedSet := performancev2.CPUSet("0")
+	profileNeedsUpdate = false
+	//if string(*initialIsolated) != string(latencyIsolatedSet) || string(*initialReserved) != string(latencyReservedSet) {
+	if *initialIsolated != latencyIsolatedSet || *initialReserved != latencyReservedSet {
+		testlog.Info(" the profile needs update ")
+		profileNeedsUpdate = true
+		profilesupdate.UpdateIsolatedReservedCpus(latencyIsolatedSet, latencyReservedSet)
 	}
 
 	if err := createNamespace(); err != nil {
-		t.Fatalf("cannot create the namespace: %v", err)
```

Comment:
yes, the namespace is created to hold the prepull daemon to make the process of pulling the pod image faster as it will be already on the nodes. but if prepull is not successful it should not block the testing because the test itself pulls the image:
  https://github.com/openshift-kni/performance-addon-operators/blob/0c9fb1fe23e10f8c89a7f8e5861662bddeb8d7cd/functests/4_latency/latency.go#L452